.github/workflows/release.yml

```diff
@@ -440,9 +452,21 @@ jobs:
         make -j$(sysctl -n hw.logicalcpu) package
         echo "::endgroup::"
 
-        # Remove the sha256 files CPack generates; we will do this ourself at
-        # the end of this workflow.
-        rm -f bundles/*.sha256
+    - name: Install gon
+      env:
+        HOMEBREW_NO_AUTO_UPDATE: 1
+        HOMEBREW_NO_INSTALL_CLEANUP: 1
+      run: |
+        brew tap mitchellh/gon
+        brew install mitchellh/gon/gon
+
+    - name: Notarize
+      env:
+        AC_USERNAME: ${{ secrets.APPLE_DEVELOPER_USERNAME }}
+        AC_PASSWORD: ${{ secrets.APPLE_DEVELOPER_PASSWORD }}
```

Comment:
I rather avoid people being confused that we user the developers username and password. You mention the username and password is app specific? Maybe better to name it as such? `APPLE_APPLICATION_USERNAME` ?

This just to avoid people misunderstanding what this is, and complaining about us not being secure or what-ever :)

Reply:
Hm, it's a username/password for the Apple Developer site, so it seemed a reasonable name to me. Can change if you would like but I don't think personally it's a cause for concern. I have mentioned in the comments in the .sh that it should ideally be an app-specific password that's used.

Reply:
Yeah, but I know our audience a bit too well sadly :P

`APPLE_DEVELOPER_SITE_USERNAME` ?

Reply:
Have replaced with `APPLE_DEVELOPER_APP_x`, hopefully that's OK.